src/main/java/net/apnic/rdap/resource/ResourceConfiguration.java

```diff
@@ -0,0 +1,57 @@
+package net.apnic.rdap.resource;
+
+import net.apnic.rdap.autnum.AutnumStatsResourceLocator;
+import net.apnic.rdap.autnum.AsnRange;
+import net.apnic.rdap.domain.Domain;
+import net.apnic.rdap.domain.DomainStatsResourceLocator;
+import net.apnic.rdap.entity.Entity;
+import net.apnic.rdap.ip.IPStatsResourceLocator;
+import net.apnic.rdap.nameserver.NameServer;
+
+import net.ripe.ipresource.IpRange;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class ResourceConfiguration
+{
+    private ResourceLocator<AsnRange> autnumResourceLocator =
```

Comment:
The AsnRange ResourceLocator makes sense. My reasoning for the other two was so I could inject the IP locator into the domain one. I didn't want to rely on autowiring here as it seems logical to me that it may not always be the case where one would want to plug in that bean.